src/content/getinvolved.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Get Involved"
+description = "Actions you can take to help support and grow the Chapel project"
+
++++
+
+## Interested in helping the Chapel project grow?
+
+Here are some ways to get involved and help nurture the Chapel
+community:
+{.content-paragraph}
+
+
+### Easy Actions
+
+- Star the project [on GitHub](https://github.com/chapel-lang/chapel)
+- Follow us on [social media](/socials)
+- Subscribe to our [YouTube channel](https://www.youtube.com/@ChapelLanguage)
+- Like, repost, and amplify Chapel content you appreciate
+- Attend weekly project and/or deep-dive [meetings](/meetings)
+- Attend the monthly educators [meeting](/meetings)
+- Ask questions on Chapel [discussion forums](/forums)
+- Invite a Chapel community member to...
+  - speak at a seminar you're involved with
+  - run a demonstration for your group
+  - take part in an interview on your blog/podcast
+- Propose some other way to get involved
+
+### Use Chapel
+
+- [Download and install](/download) Chapel
+- Compile and run sample codes:
+  - from the [`examples/` directory](https://github.com/chapel-lang/chapel/tree/main/test/release/examples) of your installation
+  - from the website's [front page](/)
+  - from the examples [page](/examples) and [repo](https://github.com/chapel-lang/ChapelExamplesAndTeachingMaterials)
+- Pair-program with a Chapel developer to explore writing code patterns of interest
+- [Learn](/learn) Chapel
+- Write your own applications in Chapel
+- Share your work with the community by
+  - mentioning it [on Discord](https://discord.gg/xu2xg45yqH)
+  - doing a 5-minute demo in our [weekly project meeting](https://github.com/chapel-lang/chapel/discussions/categories/weekly-meeting-topics)
+  - proposing a longer introduction in a [weekly deep-dive meeting](https://github.com/chapel-lang/chapel/discussions/27247)
+  - [proposing](https://github.com/chapel-lang/chapel-blog/#readme) and writing a [Chapel blog](/blog) post
+  - submitting a talk to [ChapelCon](/chapelcon)
+  - contributing the code back as a [Mason package](/docs/tools/mason/mason.html), example, or test
+ 
+
+### Help Improve Chapel
+
+- Get familiar with the [Developer Certificate of Origin](/docs/developer/bestPractices/DCO.html)
+- Improve or extend Chapel's [documentation](/docs) by editing [its sources](https://github.com/chapel-lang/chapel/tree/main/doc)
+- Improve the Chapel [website](https://github.com/chapel-lang/chapel-www)
+- Add, improve, or extend Chapel's [standard](/docs/modules/standard.html) or [package](/docs/modules/packages.html) library modules
+- Check open Chapel issues for ones labeled [good first issue](https://github.com/chapel-lang/chapel/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22) or [easy / straightforward](https://github.com/chapel-lang/chapel/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22easy%20%2F%20straightforward%22)
```

Comment:
Since we're talking about issues, should we explicitly encourage folks to report bugs they encounter as issues? It would 100% help improve the project.

Reply:
sounds good to me!